codespell_lib/data/dictionary.txt

```diff
@@ -25278,9 +25281,12 @@ stoppping->stopping
 stopps->stops
 stopry->story
 storag->storage
+storeable->storable
 storeage->storage
 stoream->stream
-storeis->stories
+storeble->storable
+storeing->storing
+storeis->stories, stores,
 storise->stories
```

Comment:
Haha, this line suddenly made me think of tortoise ! :turtle: 
No, I'm not suggesting that :grinning:

Reply:
:-D